haskell_scotty/Client.hs

```diff
@@ -59,3 +64,13 @@ instance FromJSON Client where
       (attributes >>= (.: "phone")) <*>
       (attributes >>= (.: "created_at")) <*>
       (attributes >>= (.: "updated_at"))
+  parseJSON _ = error "unexpected non-object JSON"
+
+instance QueryResults Client where
+  convertResults fs vs =
+      Client $... constructorWrap (undefined :: Client) fs vs
+
+instance QueryParams Client where
+  renderParams
+    (Client _ name' address' city' state' zip' country' email' contact' phone' created_at' updated_at') =
```

Comment:
Yes. Primes get abused in all kinds of ways. There's a lot of things one could do here with the namespace, but this seemed simplest, especially since this code should arguably be cleaned up and replaced anyhow. See the issue where I recommend we move the whole mess to `groundhog`.